scripts/parallel-nextpnr-nexus

```diff
@@ -37,7 +38,7 @@ if ! [[ ${BASH_VERSION} =~ "5.1" ]]; then
         --pdc "${PDC}" \
         --fasm "${RESULT_FASM}" \
         --device LIFCL-17-8UWG72C \
-        --seed 1
+        --seed "${SEED}"
```

Comment:
I'd be happy to use `seed_base = $((SEED * 100))` instead of setting seed_base randomly below.

(This shouldn't block merge - it's something I could do later)